packages/lab/src/pill-next/PillNext.css

```
@@ -31,64 +13,41 @@
   gap: var(--salt-spacing-50);
   padding-left: var(--salt-spacing-50);
   padding-right: var(--salt-spacing-50);
-  color: var(--pillNext-text-color);
+  color: var(--pillNext-color, var(--salt-actionable-primary-foreground));
```

```suggestion
  color: var(--saltPillNext-color, var(--salt-actionable-primary-foreground));
```
If we're adding them at all, we should update the naming